packages/backend/server/src/plugins/worker/utils/encoding.ts

```diff
@@ -35,11 +35,13 @@ export async function decodeWithCharset(
     });
   const body = await rewriter.transform(response).arrayBuffer();
 
-  if (charset) {
-    const decoder = new TextDecoder(charset);
-    res.charset = decoder.encoding;
-    return new Response(decoder.decode(body), response);
-  } else {
-    return new Response(body, response);
-  }
+  try {
+    if (charset) {
+      const decoder = new TextDecoder(charset);
+      res.charset = decoder.encoding;
+      return new Response(decoder.decode(body), response);
+    }
+  } catch {}
```

Comment:
Does the worker also have a logger feature? Print out the Warn log so you can troubleshoot issues online

Reply:
the issue here is that the TextDecoder does not define aliases for all known encodings, such as `en`, which is not an unknown error and can be safely ignored.